pallets/democracy/src/traits.rs

```diff
@@ -1,9 +1,15 @@
 use crate::{AccountVote, ReferendumIndex};
 use frame_support::dispatch::DispatchResult;
 
+pub enum Status<B> {
```

Comment:
Is this used? I run tests without it and all tests passed.

Reply:
leftover.will delete